libpysal/graph/base.py

```diff
@@ -904,7 +902,7 @@ def transform(self, transformation):
             s = self._adjacency.groupby(level=0).transform(
                 lambda group: group / math.sqrt((group**2).sum())
             )
-            nQ = self.n / s.sum()
+            nQ = self.n / s.sum()  # noqa N806
             standardized = (s * nQ).fillna(0).values  # isolate comes as NaN -> 0
```

Comment:
```suggestion
            n_q = self.n / s.sum()
            standardized = (s * n_q).fillna(0).values  # isolate comes as NaN -> 0
```

There is no reason not to follow Ruff's recommendation here.